Lib/fontTools/feaLib/ast.py

```diff
@@ -320,6 +347,7 @@ def asFea(self, indent=""):
 
 
 class LookupBlock(Block):
+    """A named lookup, containing ``statements``."""
```

Comment:
`statements` is a field of the `Block` class, so it seems redundant to call it out here specifically.

Reply:
I think I'm going to keep this. The idea is to distinguish it from a lookup reference (`lookup foo;`) which you could also describe as "a named lookup".